src/angular.prefix

```diff
@@ -1,5 +1,6 @@
-/**
- * @license AngularJS v"NG_VERSION_FULL"
+/** 
+ * @license "NG_FILENAME"
```

Comment:
Is there any specific reason why the filename is now the first item in the header?